src/util.h

```diff
@@ -136,7 +136,6 @@ namespace BCLog {
         MNSYNC      = ((uint64_t)1 << 40),
         PRIVATESEND = ((uint64_t)1 << 41),
         SPORK       = ((uint64_t)1 << 42),
-        ALERT       = ((uint64_t)1 << 43),
```

Comment:
should this be kept in some way to ensure this id isn't used again? or would that be fine?

Reply:
Should be totally fine to reuse them imo, these are local ones (unlike `inv`s for example).

Reply:
that was my thought as well, wanted to make sure my understanding was correct